i18n/nl.toml

```diff
@@ -84,64 +84,72 @@ noEnergy = "Geen meter data"
 pvExport = "Netinjectie"
 pvProduction = "Productie"
 selfConsumption = "Zelfverbruik"
+batteryTooltip = "{energy} van {total} ({soc}%)"
 
 [main.loadpoint]
 charged = "Geladen"
 duration = "Duur"
-fallbackName = "Charing point"
-power = "Snelheid"
+fallbackName = "Laadpunt"
+power = "Vermogen"
 remaining = "Resterend"
 remoteDisabledHard = "{source}: uitgeschakeld"
 remoteDisabledSoft = "{source}: adaptief PV-laden uitgeschakeld"
 
 [main.loadpointSettings]
 currents = "Laadstroom"
 default = "standaard"
-disclaimerHint = "Letop:"
-disclaimerText = "Changes zal verloren zijn nadat de server is herstart."
+disclaimerHint = "Let op:"
+disclaimerText = "Wijzigingen gaan verloren als de server herstart."
 title = "Instellingen {0}"
 vehicle = "Voertuig"
 
 [main.loadpointSettings.maxCurrent]
-label = "Max. vermogen"
+label = "Max. Stroom"
 
 [main.loadpointSettings.minCurrent]
-label = "Min. vermogen"
+label = "Min. Stroom"
 
 [main.loadpointSettings.minSoc]
 description = "Voor noodgevallen. Het voertuig wordt „snel” opgeladen tot {0}% van alle beschikbare zonne-energie, en gaat dan verder met alleen het zonneoverschot."
 label = "Min. lading %"
 
 [main.loadpointSettings.phasesConfigured]
 label = "Fasen"
-phases_0 = "auto-switching"
+phases_0 = "automatisch omschakelen"
 phases_1 = "1 fase"
 phases_1_hint = "({min} - {max})"
 phases_3 = "3 fase"
 phases_3_hint = "({min} - {max})"
 
 [main.mode]
-minpv = "Min+Solar"
+minpv = "Min+PV"
 now = "Snel"
 off = "Uit"
-pv = "Solar"
+pv = "PV"
```

Comment:
Most languages adoptes their equivalent of "sun" instead of "pv" for this naming. In German ists still PV, but well change this eventually. So try to avoid switching between both terms and embrace the ☀️